akka-docs/src/test/java/jdocs/stream/operators/sourceorflow/MapAsyncs.java

```diff
@@ -24,13 +28,13 @@ public class MapAsyncs {
   // #mapasync-concurrent
   // #mapasyncunordered
 
-  private final Source<Event, NotUsed> events =
+  private final Source<Event, NotUsed> events = // ...
+      // #mapasync-strict-order
+      // #mapasync-concurrent
+      // #mapasyncunordered
```

Comment:
This makes those samples more concise but it also makes them impossible to just copy-paste and run, maybe keep as they were? I'm on the fence.

Reply:
I honestly could have sworn that the original examples did this, but...

It's a not uncommonly-encountered question in gitter/SO that there's something excluded from the snippets that has to be included in order to actually run them.